Automattic/wp-calypso
# Stats Widget: Add 7 day highlights

Related to #75103 
## Proposed Changes

* Add the `7 Day Highlights` section.
* Apply `top-posts` and `referrers` API queries for data.

<img width="632" alt="截圖 2023-04-08 上午4 54 29" src="https://user-images.githubusercontent.com/6869813/230677782-0062964e-081c-4b90-951b-d982bf40106a.png">

## Testing Instructions



* Open your local Jetpack site.
* Check out the branch of the Jetpack to the latest `trunk`.
* Build Jetpack if necessary: `jetpack build plugins/jetpack`.
* Run `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev` under `calypso/apps/odyssey-stats`.
* Navigate to `/wp-admin/index.php?odyssey_widget=1`.
* Ensure the `7 Day Highlights` section of the Stats Widget displays in line with the design.

## Pre-merge Checklist
